apps/web/src/server/app/lpa/lpa.controller.js

```diff
@@ -12,17 +14,166 @@ import { lpaRoutesConfig as routes } from '../../config/routes.js';
+	const appealId = request.params.appealId;
+	const questionnaireData = request.session.questionnaireData;
+
+	(request.session.reviewWork ??= {}).fields = {
+		planningOfficersReport: {
+			completed: request.body['planning-officers-report-missing-or-incorrect'],
+		},
+		plansUsedToReachDecision: {
+			completed: request.body['plans-used-to-reach-decision-missing-or-incorrect'],
+			details: { value: request.body['plans-used-to-reach-decision-missing-or-incorrect-reason'] }
+		},
+		statutoryDevelopmentPlanPolicies: {
+			completed: request.body['statutory-development-plan-policies-missing-or-incorrect'],
+			details: { value: request.body['statutory-development-plan-policies-missing-or-incorrect-reason'] }
+		},
+		otherRelevantPolicies: {
+			completed: request.body['other-relevant-policies-missing-or-incorrect'],
+			details: { value: request.body['other-relevant-policies-missing-or-incorrect-reason'] }
+		},
+		supplementaryPlanningDocuments: {
+			completed: request.body['supplementary-planning-documents-missing-or-incorrect'],
+			details: { value: request.body['supplementary-planning-documents-missing-or-incorrect-reason'] }
+		},
+		conservationAreaGuidance: {
+			completed: request.body['conservation-area-guidance-missing-or-incorrect'],
+			details: { value: request.body['conservation-area-guidance-missing-or-incorrect-reason'] }
+		},
+		listedBuildingDescription: {
+			completed: request.body['listed-building-description-missing-or-incorrect'],
+			details: { value: request.body['listed-building-description-missing-or-incorrect-reason'] }
+		},
+		applicationNotification: {
+			completed: request.body['application-notification-missing-or-incorrect'],
+			details: { checkValues: checkboxDataToCheckValuesObject(request.body['application-notification-missing-or-incorrect-reason']) }
+		},
+		applicationPublicity: {
+			completed: request.body['application-publicity-missing-or-incorrect'],
+		},
+		representations: {
+			completed: request.body['representations-missing-or-incorrect'],
+			details: { value: request.body['representations-missing-or-incorrect-reason'] }
+		},
+		appealNotification: {
+			completed: request.body['appeal-notification-missing-or-incorrect'],
+			details: { checkValues: checkboxDataToCheckValuesObject(request.body['appeal-notification-missing-or-incorrect-reason']) }
+		}
+	};
+
+	const {
+		body: { errors = {}, errorSummary = [] }
+	} = request;
+
+	if (Object.keys(errors).length > 0) {
+		for (const key in errors) {
+			if (errors.hasOwnProperty(key)) {
+				// eslint-disable-next-line unicorn/consistent-destructuring
+				request.session.reviewWork.fields[camelCase(key.replace('-missing-or-incorrect-reason', ''))].details.error = { msg: errors[key].msg };
+			}
+		}
+
+		return response.render(routes.reviewQuestionnaire.view, {
+			backURL: `${routes.home.path}?direction=back`,
+			errors,
+			errorSummary,
+			questionnaireData,
+			// eslint-disable-next-line unicorn/consistent-destructuring
```

Comment:
Maybe this rule is too much. Please change it to a warning maybe in our eslint config and remove all this disable lines.

Reply:
Very happily done! 

Personally I think most of these `unicorn` rules are unnecessary. They are very opinionated (for example https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-reduce.md) and just serve to slow down development. Often the reasoning given for the rule is brief and vague and I don't see what makes [sindresorhus](https://github.com/sindresorhus) an authority on what parts of the javascript language should be summarily consigned to the dustbin.

I will admit the "prevent abbreviations" rule is a nice reminder. Just to show I'm not completely against it 😂 

Apologies for the complaining tone, I just feel quite strongly on the subject of linters and what I consider to be their rampant overuse!